tests/integration/conftest.py

```diff
@@ -612,7 +646,7 @@ def qtest_requirements(self):
         """A list of qTest swagger_client requirement models associated with the given test case.
 
         Returns:
-            list(swagger_client.RequirementResource)
+            [swagger_client.RequirementResource]
```

Comment:
The JetBrains IDE does not recognize "list(*)" as a valid docstring data type. Fixed.